include/taskomat/deserialize_sequence.h

```diff
@@ -51,29 +51,29 @@ std::istream& operator>>(std::istream& stream, Step& step);
  * the file does not exist.
  *
  * To deserialize a Step it must consist with following minimum properties:
- * /code
+ * \code
  * -- type: action \a or if \a or ...
  * -- label: < \a label \a description >
  * /endcode
```

Comment:
If I look at line 76, this should maybe also changed, along with line 65?

Reply:
Ooops. Good point. Will do.

Reply:
Done.